src/Symfony/Component/Notifier/Bridge/Smsc/SmscTransport.php

```diff
@@ -30,11 +30,11 @@ final class SmscTransport extends AbstractTransport
 {
     protected const HOST = 'smsc.ru';
 
-    private $login;
-    private $password;
-    private $from;
+    private ?string $login;
+    private ?string $password;
+    private string $from;
 
-    public function __construct($username, $password, $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    public function __construct(?string $username, ?string $password, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
Any reason why we did not have types here? 🤔

Reply:
none I suppose, let's fix that on 5.4 :)